update_v1.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+
+set -x
+set -e
+
+# Remove current V1
+rm -rf pydantic/v1
```

Comment:
IMO, it would be better to move this right before the `cp` below. This way if something goes wrong with the clone and/or checkout we will still have the files.

It doesn't matter that much as we still can do `git reset --hard` anyway. So, up to you.

Reply:
Done!